website/docs/artist_hosts_nuke_tut.md

```diff
@@ -0,0 +1,195 @@
+---
+id: artist_hosts_nuke_tut
+title: Nuke QuickStart
+sidebar_label: Nuke QuickStart
+---
+
+This QuickStart is just a small introduction to what OpenPype can do for you. It attempts to make an overview for compositing artists, and simplifies processes that are better described in specific parts of the documentation.
+
```

Comment:
Let's take what's in the Nuke page and use it as a category in here, so we have a single page with all things nuke. First a quick overview of nuke specific tools, then this quickstart explanation.

Reply:
Done. I kept links for global tools as well.